admin/templates/configuration/tmp/configure/sql/CssData.sql

```diff
@@ -0,0 +1,33 @@
+-- MySQL Script generated by MySQL Workbench
+-- Tue Jul 28 15:39:09 2015
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+-- -----------------------------------------------------
+-- Schema qservCssData
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema qservCssData
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `qservCssData` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `qservCssData` ;
+
+-- -----------------------------------------------------
+-- Table `qservCssData`.`kvData`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `qservCssData`.`kvData` (
+  `kvKey` CHAR(255) NOT NULL COMMENT 'slash delimited path',
+  `kvVal` TEXT(1024) NULL COMMENT 'value for key',
+  `parentKey` CHAR(255) NULL COMMENT 'slash delimited path of parent key',
+  PRIMARY KEY (`kvKey`)  COMMENT '')
```

Comment:
We should probably add foreign key constraint for parent key here to enforce referential integrity. I'd also think of defining `kvId` integer auto-increment column to be used as primary key, so that `parentKey` refers to that instead of `kvKey` column. `kvKey` then has to be made unique.